lib/main_publiccloud.pm

```diff
@@ -18,15 +18,23 @@ our @EXPORT = qw(
   load_publiccloud_tests
 );
 
+sub load_3rd_party_image_test {
```

Comment:
Well why we just don't export that subroutine from `main_containers`? :disappointed: